pom.xml

```diff
@@ -832,8 +556,8 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.4</version>
```

Comment:
version is 2.5.3 in parent pom. Not necessarily a problem but might be worth checking?